crates/biome_aria/src/roles.rs

```diff
@@ -1241,6 +1241,60 @@ impl<'a> AriaRoles {
+            "alertdialog" => &AlertDialogRole as &dyn AriaRoleDefinitionWithConcepts,
+            "cell" => &CellRole as &dyn AriaRoleDefinitionWithConcepts,
+            "columnheader" => &ColumnHeaderRole as &dyn AriaRoleDefinitionWithConcepts,
+            "definition" => &DefinitionRole as &dyn AriaRoleDefinitionWithConcepts,
+            "figure" => &FigureRole as &dyn AriaRoleDefinitionWithConcepts,
+            "form" => &FormRole as &dyn AriaRoleDefinitionWithConcepts,
+            "graphics-document" => &GraphicsDocumentRole as &dyn AriaRoleDefinitionWithConcepts,
+            "graphics-object" => &GraphicsObjectRole as &dyn AriaRoleDefinitionWithConcepts,
+            "grid" => &GridRole as &dyn AriaRoleDefinitionWithConcepts,
+            "gridcell" => &GridCellRole as &dyn AriaRoleDefinitionWithConcepts,
+            "group" => &GroupRole as &dyn AriaRoleDefinitionWithConcepts,
+            "img" => &ImgRole as &dyn AriaRoleDefinitionWithConcepts,
+            "link" => &LinkRole as &dyn AriaRoleDefinitionWithConcepts,
+            "list" => &ListRole as &dyn AriaRoleDefinitionWithConcepts,
+            "listbox" => &ListBoxRole as &dyn AriaRoleDefinitionWithConcepts,
+            "listitem" => &ListItemRole as &dyn AriaRoleDefinitionWithConcepts,
+            "navigation" => &NavigationRole as &dyn AriaRoleDefinitionWithConcepts,
+            "row" => &RowRole as &dyn AriaRoleDefinitionWithConcepts,
+            "rowgroup" => &RowGroupRole as &dyn AriaRoleDefinitionWithConcepts,
+            "rowheader" => &RowHeaderRole as &dyn AriaRoleDefinitionWithConcepts,
+            "search" => &SearchboxRole as &dyn AriaRoleDefinitionWithConcepts,
+            "searchbox" => &SearchboxRole as &dyn AriaRoleDefinitionWithConcepts,
+            "table" => &TableRole as &dyn AriaRoleDefinitionWithConcepts,
+            "term" => &TermRole as &dyn AriaRoleDefinitionWithConcepts,
+            "textbox" => &TextboxRole as &dyn AriaRoleDefinitionWithConcepts,
+            "generic" => &GenericRole as &dyn AriaRoleDefinitionWithConcepts,
+            "caption" => &CaptionRole as &dyn AriaRoleDefinitionWithConcepts,
+            "main" => &MainRole as &dyn AriaRoleDefinitionWithConcepts,
+            "time" => &TimeRole as &dyn AriaRoleDefinitionWithConcepts,
+            "p" => &ParagraphRole as &dyn AriaRoleDefinitionWithConcepts,
+            "aside" => &ComplementaryRole as &dyn AriaRoleDefinitionWithConcepts,
+            "blockquote" => &BlockQuoteRole as &dyn AriaRoleDefinitionWithConcepts,
+            "associationlist" => &AssociationListRole as &dyn AriaRoleDefinitionWithConcepts,
+            "status" => &StatusRole as &dyn AriaRoleDefinitionWithConcepts,
+            "contentinfo" => &ContentInfoRole as &dyn AriaRoleDefinitionWithConcepts,
+            "region" => &RegionRole as &dyn AriaRoleDefinitionWithConcepts,
+            _ => return None,
+        };
+
+        Some(role_candidate.concepts_by_role().unwrap())
```

Comment:
```suggestion
       role_candidate.concepts_by_role()
```

Remember to never use `unwrap`, it's very unsafe. Use it only when you are 100% certain that is safe. And when you know that it's safe, it's customary to add a `// SAFETY:` comment.

Look for `// SAFETY` in our code base for examples. 

In this example, it is very unsafe, because that's a public API that can be called with any arbitrary string, so a developer can pass `"bar"` and the code will panic.